src/rules/NoIgnoredReturnRule.ts

```diff
@@ -0,0 +1,240 @@
+import * as Lint from "tslint";
+import { IOptions, IRuleMetadata, Utils } from "tslint";
```

Comment:
You don't need the second import, because you've already imported everything in the first one. Just use `Lint. IOptions` for example.